src/control.f90

```diff
@@ -824,6 +826,23 @@ subroutine md_run (fixed_potential, vary_mu, total_energy)
 
     do iter = i_first, i_last ! Main MD loop
        mdl%step = iter
+       
+       ! At a given time step, if the difference between T_ext and the final temperature is larger than 
+       ! the temperature step (with 5% margin), then update T_ext 
+       ! Temperature evolves linearly from temp_ion to temp_ion_end over MDn_steps steps
+       ! Stop when target temperature has been reached
+       if  (abs(mdl%T_ext - temp_ion_end) > 0.95*abs((temp_ion_end-temp_ion)/dble(MDn_steps))) then
```

Comment:
I wonder if we should make this fraction of 0.95 a parameter that can be set: presumably it acts as some kind of thermal inertia/

Reply:
Actually, it is a (possibly clumsy) way to check at a given time step whether we have reached the target final temperature or not. 
With float numbers, setting the condition at exactly the temperature step, or to zero, it happened that the condition was not properly fulfilled and the cooling/heating continued beyond the target temperature.